Library/Homebrew/cask/lib/hbc/artifact/abstract_uninstall.rb

```diff
@@ -267,14 +267,17 @@ def trash_paths(*paths, command: nil, **_)
                 set trashedItem to POSIX path of (item i of trashedItems as string)
                 set output to output & trashedItem
                 if i < count trashedItems then
-                  set output to output & (do shell script "printf \"\\0\"")
+                  set output to output & character id 0
                 end if
               end repeat
 
               return output
             end tell
           end run
         EOS
+
+        # Remove AppleScript's automatic newline.
+        result.tap { |r| r.stdout.sub!(/\n$/, "") }
```

Comment:
`chomp!` doesn't work?

Reply:
`chomp` doesn't work on the `\r\n` edge case.